assets/evaluation_results/squad_v2_gpt-35-turbo-0613_question_answering/spec.yaml

```diff
@@ -0,0 +1,33 @@
+type: evaluationresult
+name: squad_v2_gpt-35-turbo-0613_question_answering
+version: 2.10.05
+display_name: squad_v2_gpt-35-turbo-0613_question_answering
+description: gpt-35-turbo-0613 run for squad_v2 dataset
+dataset_family: squad_v2
+dataset_name: squad_v2
+
+model_name: gpt-35-turbo-0613
+model_version: "0613"
```

Comment:
Model version should be 2 as per model asset

Reply:
Hi, we have added the version for squadv2 dataset similar to the old results as in truthfulqa-generation dataset.

![image](https://github.com/Azure/azureml-assets/assets/38103118/7bf9e6f7-3835-43b7-8e96-3a1de20aacab)

can you please let me know if you suggest to update the version to 2?